src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPackageStep/help-artifactsPath.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Path to artifact directory relative to workspace. Jenkin's environment variables can be used to specify this value.
```

Comment:
Path to the artifact directory, relative to the workspace. Jenkins environment variables can be used to specify this value.